lib/mongodb/queries.nit

```diff
@@ -70,10 +70,25 @@ import mongodb
 class MongoMatch
 	super JsonObject
 
-	private fun op(name: String, field: String, value: nullable Jsonable): MongoMatch do
-		var q = new JsonObject
-		q["${name}"] = value
-		self[field] = q
+	# Define a custom operator for `field`
+	#
+	# If no `field` is specified, append the operator the the root object:
+	# ~~~json
+	# {<name>: <value>}
+	# ~~~
+	#
+	# Else, append the operator to the field:
+	# ~~~json
+	# {field: {<name>: <value>} }
+	# ~~~
```

Comment:
The documentation is misleading. It does not mention that an `$` is prepended to the operator name. There should be a hint at least in the snippets. Exemple: `{"<field>": {"$<name>": <value>}}`